tests/cucumber/steps/steps.js

```diff
@@ -2143,35 +2473,39 @@ module.exports = function getSteps(options) {
   When(
     'we make a Lookup Block call against round {int}',
     async function (round) {
-      await this.indexerClient.lookupBlock(round).do();
+      await doOrDoRaw(this.indexerClient.lookupBlock(round));
```

Comment:
The old syntax with a `do()` at the end looks nicer, but assume this was needed for a good reason.

Reply:
Yes, this is necessary because we run some requests against a mock server that always returns an empty response with the intent to ensure that the right path is requested.

Now that indexer has typed responses, using the `do()` method will attempt to parse a response and fail. We have a `doRaw()` method which doesn't parse, which is ideal for this case, and the `doOrDoRaw` function choose the right one based on the specific test.